core/src/main/scala/com/nvidia/spark/rapids/tool/EventLogPathProcessor.scala

```diff
@@ -288,23 +296,55 @@ object EventLogPathProcessor extends Logging {
       } else {
         filteredByMinSize
       }
+      // filter by date time range first
```

Comment:
I think there is much redundant code in this part and `filteredByMinSize` and `filteredByMaxSize` sections as well. Can we deduplicate these?